docs/streams/upgrade-guide.html

```diff
@@ -134,6 +134,12 @@ <h3 class="anchor-heading"><a id="streams_notable_changes" class="anchor-link"><
     </p>
 
     <h3><a id="streams_api_changes_370" href="#streams_api_changes_370">Streams API changes in 3.7.0</a></h3>
+    <p>
+        We added a new method to <code>KafkaStreams</code>, namely <code>KafkaStreams#setStandbyUpdateListener()</code> in
+        <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-988%3A+Streams+Standby+Update+Listener">KIP-988</a>.
+        In which users can provide their customized implementation of the org.apache.kafka.streams.processor.StandbyUpdateListener interface to continuously monitor changes to standby tasks.
```

Comment:
> In which users can provide their customized implementation...
This is a subordinate clause, not an independent clause. I would propose replacing the `.` after `</a>` with a `,` and replacing `In` with `in`.

Reply:
+1 to Colt's suggestion, otherwise LGTM